hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/bucket/BucketCache.java

```diff
@@ -359,24 +373,22 @@ public String getIoEngine() {
 
   /**
    * Get the IOEngine from the IO engine name
-   * @param ioEngineName
-   * @param capacity
    * @return the IOEngine
    * @throws IOException
    */
-  private IOEngine getIOEngineFromName(String ioEngineName, long capacity)
+  private IOEngine getIOEngineFromName()
```

Comment:
See below comment also. Actually there is no need to have any change here now. We do not need to pass any extra args to FileIOE constructor now.  No need to even change the order of call to getIOEngineFromName() above right. Better keep only the req change IMHO.